src/objective.jl

```diff
@@ -491,11 +493,103 @@ function _set_objective_coefficient(
     return
 end
 
+"""
+    set_objective_coefficients(
```

Comment:
```suggestion
    set_objective_coefficient(
```

Shouldn't we just keep the same name and provide a new method?

Reply:
That is certainly an option. I would be totally fine with it. It might even reduce the number of typos in the code, with which I suffered myself.

Reply:
Yeah. We have dispatch. Let's use it instead of adding more functions.